docs/dictionary/command/read-from-process.lcdoc

```diff
@@ -26,41 +26,39 @@ Example:
 read from process myProcess for 10 int4s in 250 milliseconds
 
 Parameters:
-processName:
+processName (string):
 The processName specifies the name and location of the process you want
 to read from. It must be the same as the path you used with the open
-process command.>*Important:* The processName is case-sensitive, even on
-platforms where file names are not case-sensitive. It must be exactly
-the same--including the case of characters--as the name you used with
-the open processcommand.
-
-start:
-The start specifies the character or byte position in the process output
-where you want to begin reading.
-
-strin:
-
+process command.
+>*Important:*  The <processName> is <case-sensitive>, even on
+> <platform|platforms> where file names are not <case-sensitive>. It
+> must be exactly the same--including the case of characters--as the
+> name you used with the <open process> <command>.
 
-amoun:
 
+start (string):
+The <start> specifies the <character> or <byte> position in the
+<process> output where you want to begin reading.
 
-chunkType:
-One of chars, characters, words, items, lines, int1, uInt1, int2, uint2,
-int4, or uint4. The read from process command reads amount of the
-specified chunkType. If you don't specify a chunkType, amount characters
-are read.
+chunkType (enum):
```

Comment:
You should remove `(enum)` from this for now, as I don't think this entry is best formatted in the enum style.